PhpRbac/src/PhpRbac/core/lib/rbac.php

```diff
@@ -775,6 +775,27 @@ function roles($Permission, $OnlyIDs = true)
     		    WHERE PermissionID=? ORDER BY TP.ID", $Permission );
 		}
 	}
+	/**
+	 * Returns all users with a permission
+	 *
+	 * @param integer $Permission
+	 *        	ID
+	 * @return Array 1D or null
+	 */
+	function users($Permission)
```

Comment:
function getUsersByPermissionId($PermissionId)

Reply:
this is not the way other functions in a library named

Reply:
that's clear function name, belongs to standard naming